libs/cart/state/src/selectors/cart-item-entities/cart-item-entities.selectors.ts

```diff
@@ -142,6 +183,80 @@ const createCartItemEntitiesSelectors = <
 		}
 	)
 
+  const selectCartItemQuantity = createSelector(
+    selectCartItemEntities,
+    (cartItems, props) => {
+      const cartItem: DaffCartItem = selectCartItem.projector(cartItems, { id: props.id });
```

Comment:
I think you can do: `selectCartItem.projector(cartItems, { id: props.id })?.qty || 0` instead. So you should be able to remove a lot of the new `const`s in these selectors

Reply:
changed